ui-particles-angular/projects/ui-policy-studio-angular/src/lib/components/flow-form-dialog/flow-proxy-form-dialog/gio-ps-flow-proxy-form-dialog.harness.ts

```diff
@@ -53,19 +62,52 @@ export class GioPolicyStudioFlowProxyFormDialogHarness extends ComponentHarness
       await pathOperatorInput.clickOptions({ text: new RegExp(flow.pathOperator, 'i') });
     }
     if (flow.path) {
-      await pathInput.setValue(flow.path);
+      const input = await this.pathInput();
+      await input.setValue(flow.path);
     }
     if (flow.methods) {
+      const methodsInput = await this.methodsInput();
       await methodsInput.removeTag('ALL');
       for await (const method of flow.methods) {
         await methodsInput.addTag(method);
       }
     }
     if (flow.condition) {
+      const conditionInput = await this.conditionInput();
       await conditionInput.setValue(flow.condition);
     }
   }
 
+  public async getFlowFormValues(): Promise<GioPolicyStudioFlowProxyFormData> {
```

Comment:
GioPolicyStudioFlowProxyFormData is not exactly good because for ex entrypoint have not the same value as the display value same for operator. (display name vs value id)
but as the both are string 🤷‍♂️ it's work. 
IMHO i préfère a type only for this special harness return.

Juste a note. good for me like this ;)

Reply:
I think the issue is the name of the type because it is not the form data but the specific type created in the harness. I will think of a different name